### PR TITLE
Modify web search

### PR DESCRIPTION
**Pull request by aansaarii**
This PR updates the web-search benchmark. The highlight of the changes is as follows:
1. The Solr engine is updated from 6.6 to 9.0. 
2. The index file is changed. The old one was based on Lucene 5. The new index is based on Lucene 9. 
3. Some minor modifications to the client image.

### CODE REVIEW

File: docs/benchmarks/web-search.md [hunk 1]
```diff
@@ -12,47 +12,65 @@ The Web Search benchmark relies on the [Apache Solr][apachesolr] search engine f
...
-These images are automatically built using the mentioned Dockerfiles available on [`https://github.com/parsa-epfl/cloudsuite/tree/master/benchmarks/web-search`][repo].
+These images are automatically built using the mentioned Dockerfiles available [`here`][repo].
 
 ### Starting the dataset ###
 
-Downloads the dataset index
+The following command downloads and mounts the dataset index:
 
-	$ docker run cloudsuite/web-search:dataset
+  $ docker run --name web_search_dataset cloudsuite/web-search:dataset
```
Comment (Xusine1131) [inline docs/benchmarks/web-search.md:26]: Can you format all the terminal commands using the way below?

```
$ docker run --name web_search_dataset cloudsuite/web-search:dataset
```

Just using three \` to create a code block.

File: docs/benchmarks/web-search.md [hunk 2]
```diff
@@ -12,47 +12,65 @@ The Web Search benchmark relies on the [Apache Solr][apachesolr] search engine f
...
+  $ Index Node IP Address: 192.168.1.47
 
 ### Starting the client and running the benchmark ###
 
 To start a client you have to first `pull` the client image and then run it. To `pull` the client image, use the following command:
 
-	$ docker pull cloudsuite/web-search:client
+  $ docker pull cloudsuite/web-search:client
 
 The following command will start the client node and run the benchmark. The `server_address` refers to the IP address, in brackets (e.g., "172.19.0.2"), of the index node that receives the client requests. The four numbers after the server address refer to: the scale, which indicates the number of concurrent clients (50); the ramp-up time in seconds (90), which refers to the time required to warm up the server; the ramp-down time in seconds (60), which refers to the time to wait before ending the benchmark; and the steady-state time in seconds (60), which indicates the time the benchmark is in the steady state. Tune these parameters accordingly to stress your target system.
```
Comment (Xusine1131) [inline docs/benchmarks/web-search.md:48]: I think it's better to give the command prototype first, like:
```
$ docker run -it --name web_search_client --net host cloudsuite/web-search:client <server_address> <number of concurrent clients>  <The ramp up time in secod> <the ramp down time in second> <measurement time in second>

And then, give your example by saying "Here is an example where we use 50 clients, 90-second warm up time, 60-second ramp-down time and 60-second measurement time"
```

**Review by Xusine1131 — CHANGES_REQUESTED** [review]

**Comment by Xusine1131** [comment]
Also, can you please fix the spell check error by adding new words to the `.wordlist`?

File: benchmarks/web-search/client/files/search/deploy/run.xml~ [hunk 3]
```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (aansaarii) [reply benchmarks/web-search/client/files/search/deploy/run.xml~:1]: It was there by default. I think we can remove it.

File: benchmarks/web-search/client/files/search/distributions/Ziphian.java [hunk 4]
```diff
@@ -0,0 +1,217 @@
+/* CloudSuite4.0 Benchmark Suite
```
[the remark this answers is not in the document]
Comment (aansaarii) [reply benchmarks/web-search/client/files/search/distributions/Ziphian.java:1]: The Dockerfile specifies the distribution [here](https://github.com/parsa-epfl/cloudsuite/blob/modify_web_search/benchmarks/web-search/client/Dockerfile#L13). Then, the requested distribution is copied to the build directory [here](https://github.com/parsa-epfl/cloudsuite/blob/modify_web_search/benchmarks/web-search/client/docker-entrypoint.sh#L22). BTW, I have not tested the other distribution that is `Random.java`.

**Comment by aansaarii** [comment]
Thanks for your feedback @Xusine1131.
Please check the latest pushes.

File: benchmarks/web-search/client/files/search/distributions/Ziphian.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (Xusine1131) [reply benchmarks/web-search/client/files/search/distributions/Ziphian.java:1]: Should we mention that we can have a different distribution in the document? I think the distribution is important for doing research.

**Review by Xusine1131 — APPROVED** [review]

**Pull request merged**